src/components/Search/index.tsx

```diff
@@ -0,0 +1,73 @@
+import Box from '@mui/material/Box';
+import TextField from '@mui/material/TextField';
+import './style.scss';
+import React, { useState, useEffect } from 'react';
+import Autocomplete from '@mui/material/Autocomplete';
+import axios from 'axios';
+
+function Search() {
+    const [search, setSearch] = useState('');
+    const [error, setError] = useState(null);
+    const [isLoaded, setIsLoaded] = useState(false);
+    const [items, setItems] = useState([]);
+
+    useEffect(() => {
+        if (search !== '') {
```

Comment:
you can use '!search' instead of search=''

Reply:
Maybe you mean "search" because if search = '' ", "!search" returns true? :wink: